### DISCUSSION TITLE
When should you use "Merge playlist to play as a single track"?

### DISCUSSION THREAD

**Discussion by heathcliff2019** [AzuraCast/AzuraCast#7496] [Q&A]
I run a daily countdown top 10 and I am not sure to check "Merge playlist to play as a single track" or not. When should you use this function?

**Comment by silvaalex2021** [comment]
If your countdown is in a single file, there’s no need to use this function. However, if your countdown or any other show is split into multiple files and you want to ensure they play without anything else interrupting them, it’s a good idea to use this function.

**Reply by heathcliff2019** [reply]
Thanks @silvaalex2021, I use the songs from the media library and between each song a short stinger. You recommend to merge this playlist to play as a single track? 

When using this function, will it use the name of the playlist instead?

**Reply by silvaalex2021** [reply]
I do exactly that on my radio station. I have my shows in the media library, along with jingles and commercials. I create the show’s playlist with the relevant content, schedule it, and select this option to play everything as a single track.

Regarding your second question, the answer is no. What this function does is build a queue with all the playlist content at once, but what listeners see are still the metadata of each individual file being played. If you're interested, try scheduling a playlist like this for a specific time, and when that time comes, go to the upcoming songs queue in AzuraCast to get an idea of how it's done...

**Reply by heathcliff2019** [reply]
I will try it out tomorrow. Thank you for sharing your knowledge with me I sincerely appreciate it @silvaalex2021

**Reply by heathcliff2019** [reply]
As promised @silvaalex2021, so it did exactly that what you have explained but it had issues with sticking it to the order of the audio files. It played 'sometimes' two short stingers after each other.

Did you experienced this issue too and, if so, how did  you solved it?

**Reply by heathcliff2019** [reply]
Hi @silvaalex2021, I looked into the log file and I think I found the cause of the issue. It seems one of the tracks in my top 10 playlist had an attached album cover in the **webp** format, which might be causing problems. The log showed that AzuraCast had trouble processing this image, and that could be why the stingers sometimes play incorrectly or back-to-back.

Here’s the specific error from the log:

"Unknown attached picture mimetype: image/webp, skipping."

To fix this, I’m going to replace the album cover image with a jpg version and see if that resolves the issue. If it turns out Azura can’t handle tracks with webp album covers, this might be the root cause. I’ll let you know how it goes!

**Reply by heathcliff2019** [reply]
Hey @silvaalex2021, quick update on the stinger issue I reported earlier. After digging into the Liquidsoap log file, I found that it wasn’t just the webp format causing the issue. Some tracks had album covers in JPG format **but with a resolution of 96DPI** instead of the standard 72DPI, which was triggering problems in AzuraCast.

I’ve gone through and fixed the following tracks by replacing their covers.

To troubleshoot, I checked the Liquidsoap log. You can find the relevant info by searching for "Unknown attached picture mimetype": This indicates that Liquidsoap couldn't handle the album cover file, often because of an unsupported format like webp or unusual metadata.

Check for logs around the time of playback issues, as DPI/resolution errors don't directly appear, but playback inconsistencies (like double stingers or songs never played) can signal album cover problems. Files with higher DPI (96 instead of 72) and WEBP instead of JPG seem to cause trouble.

After replacing the images with 72DPI and or JPG versions, everything seems to be running smoothly again.

Time will tell tomorrow when I listen to the top 10 again.

**Reply by LordHelmchen666** [reply]
> Hey @silvaalex2021, quick update on the stinger issue I reported earlier. After digging into the Liquidsoap log file, I found that it wasn’t just the webp format causing the issue. Some tracks had album covers in JPG format **but with a resolution of 96DPI** instead of the standard 72DPI, which was triggering problems in AzuraCast.
> 
> I’ve gone through and fixed the following tracks by replacing their covers.
> 
> To troubleshoot, I checked the Liquidsoap log. You can find the relevant info by searching for "Unknown attached picture mimetype": This indicates that Liquidsoap couldn't handle the album cover file, often because of an unsupported format like webp or unusual metadata.
> 
> Check for logs around the time of playback issues, as DPI/resolution errors don't directly appear, but playback inconsistencies (like double stingers or songs never played) can signal album cover problems. Files with higher DPI (96 instead of 72) and WEBP instead of JPG seem to cause trouble.
> 
> After replacing the images with 72DPI and or JPG versions, everything seems to be running smoothly again.
> 
> Time will tell tomorrow when I listen to the top 10 again.

Did you report this to Liquidsoap developers? If not, would you please consider do it. Thanks.

**Reply by heathcliff2019** [reply]
@LordHelmchen666 no but I will be doing that tomorrow. If my countdown top 10 works, so no skipping track anymore, I know it for sure and will report it.

**Reply by heathcliff2019** [reply]
@LordHelmchen666 this issue is no longer going on after changing removing the current cover from the meta data. Changed the cover;s  DPI from 96 to 72, saving the cover as jpg and re-attaching it to the meta data. 

I also learned that Azuracast change the cover's DPI back to 96. This is totally unnecessary and takes extra resources. Web images especially that size should be standard in 72 DPI.

**Comment by silvaalex2021** [comment]
I encountered this type of problem, but it was a configuration error—the playlist was set to play in "shuffle" mode. The system will follow the playlist order as well as its playback configuration. I suggest checking the playlist order and making sure it is set to "sequential."

**Reply by heathcliff2019** [reply]
Hi @silvaalex2021 and how was your day?

I checked and the playlist was set correctly on 'Sequential'

**Reply by silvaalex2021** [reply]
Well, in that case, a final option is to check the playlist order. On the playlist page, find the playlist you want to modify and click the "more" button, located to the right of the red "delete" button. Then, click on "reorder" and check if everything is in the order you want. If this doesn’t work, then we have a problem...

**Reply by heathcliff2019** [reply]
Hello again @silvaalex2021 yes the playlist order is set correctly too.
